docs/OCP_CI_Tutorials/Scenarios/Provision_Guide.md

```diff
@@ -66,7 +82,9 @@ tests:
       BASE_DOMAIN: aws.interop.ccitredhat.com
     workflow: aws-ipi
 ```
+
 - **product_name**: The shortname of the product under test.
```

Comment:
Since we define this earlier in this document, it it necessary to define it here as well?

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)